packages/sdks/angular-sdk/projects/angular-sdk/src/lib/components/access-key-management/access-key-management.component.spec.ts

```diff
@@ -19,6 +19,7 @@ describe('DescopeAccessKeyManagementComponent', () => {
   let fixture: ComponentFixture<AccessKeyManagementComponent>;
   let mockedCreateSdk: jest.Mock;
   const onSessionTokenChangeSpy = jest.fn();
+  const onIsAuthenticatedChangeSpy = jest.fn();
```

Comment:
Is it relevant for widgets as well?

Reply:
we don't, also don't need `onSessionTokenChangeSpy` so removing them both 👍